sendgrid/helpers/exceptions/exceptions.py

```diff
@@ -0,0 +1,3 @@
+class SendGridException(Exception):
+    def __init__(self, *args, **kwargs):
```

Comment:
Here is an example of how it was done previously: https://github.com/sendgrid/sendgrid-python/blob/0942f9de2d5ba5fedb65a23940ebe1005a21a6c7/sendgrid/exceptions.py